assets/src/js/global.js

```diff
@@ -1,3 +1,19 @@
+/* eslint-env es6 */
+/* eslint-disable */
+
 /**
  * Generic site JavaScript.
  */
+
+/**
+ * Search for Donation Button and add parameter link Op1.
+ */
+document.querySelectorAll('[href^="https://donate.wikimedia.org"]').forEach( function( item ) {
```

Comment:
Now that the theme is in its own repo we should revisit how to integrate this into the main JS bundle, possibly by moving it to `src/scripts/modules` and including it from `scripts/shiro.js`

Reply:
Updated in [9bb40a2](https://github.com/wikimedia/shiro-wordpress-theme/pull/17/commits/9bb40a25747d9e40149eb8bd5825aeafccda3636).